index.html

```diff
@@ -14,8 +14,16 @@
         </header>
         <main>
             <section class="main-content">
-                <div class="worker-container"></div>
-                <div class="client-container"></div>
+                <div class="worker-container">
+                    <div class="main-logo">
+                        <img src="./img/logo-img.png" class="logo-img">
+                        <img src="./img/logo-img2.png" class="logo-img2">
```

Comment:
もう少しclass名をこだわってもいいのかなと思いました:eyes:
``logo-img2``だと何の要素か分かりづらいと思ったのでもう少し説明的な名前でもいいのかなと思いました。

これはnitsなので修正する必要はないのですが、imgの名前も安易かな？と感じました。(imgなのは明らかなので名前にimgは不要かと)

下記はsugestionですがこれが正解なわけではなくここらへんも好みでなので、最初に自分でルールを決めておくといいと思います。

```suggestion
                        <img src="./img/main-logo.png" class="main-logo">
                        <img src="./img/main-logo-sub.png" class="main-logo-sub">
```

Reply:
MUST
このような命名が他の開発者の命名規則のルールになり、やがて収拾つかなくなるので

Reply:
コメントありがとうございます！
今後も気を付けます！
修正いたしました！

8792d76